module.nix

```diff
@@ -38,6 +38,11 @@ in {
                     default = "/sys/devices/pci0000:00/0000:00:18.3/hwmon/hwmon2/temp1_input";
                     description = "path to the cpu thermal sensor, probably something in /sys/device";
                 };
+                GPUThermalZone = mkOption {
+                    type = types.nullOr types.str;
+                    default = "";
+                    description = "path to the gpu thermal sensor, probably something in /sys/device";
```

Comment:
Please also mention here that this is AMD specific and that the path is relative to ```/sys/class/drm/<card>/```

Reply:
Yeah lol I totally forgot about nvidia